routes/definitions.js

```diff
@@ -37,6 +47,9 @@ async function getDefinitions(request, response) {
   if (pattern) return response.send(await definitionService.suggestCoordinates(pattern))
   if (!validator.validate('definitions-find', request.query)) return response.status(400).send(validator.errorsText())
   const result = await definitionService.find(request.query)
+  if (Array.isArray(result.data)) {
+    response.set('cache-tag', result.data.map(project_tag).join())
```

Comment:
Is this set up so that when any of the definitions that come back as a result of the `find()` get invalidated the cache of the query is purged?

Reply:
Correct. You get a number of items with coordinates from find; the result is tagged with cache tags, one per item returned. Then when any of those definitions is changed, the CDN is asked to invalidate any and all results containing said tag. I believe it’s quite good

Reply:
👍 awesome

Reply:
do we need to filter out the null's that come back from project_tag?

Reply:
feedback incorporated at https://github.com/clearlydefined/service/pull/503/commits/ae49039da28a6fcaaa8e2c39f99df4c1a35ba41e